application/org.openjdk.jmc.flightrecorder.flameview-java/src/main/resources/org/openjdk/jmc/flightrecorder/flameviewjava/messages_ja.properties

```diff
@@ -0,0 +1,57 @@
+#
+#  Copyright (c) 2022, Datadog, Inc. All rights reserved.
```

Comment:
Needs to be an Oracle/Datadog dual copyright. Check other Datadog provided files for examples.